packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget-components.tsx

```diff
@@ -78,24 +82,28 @@ export const TransformWidgetInput: FunctionalComponent<
   };
 
   const handleKeyDown = (event: KeyboardEvent): void => {
-    event.stopPropagation();
+    const commandOrControlModifier = event.ctrlKey || event.metaKey;
 
     if (event.key === 'ArrowUp') {
       onIncrement?.();
     } else if (event.key === 'ArrowDown') {
       onDecrement?.();
+    } else if (event.key === 'z' && commandOrControlModifier) {
+      event.preventDefault();
+
+      onUndo?.();
```

Comment:
in what scenarios is `onUndo` not defined? Should we throw an exception or warning if it's not defined and we're trying to do the keybind?

Reply:
Yeah, this could probably just always be defined. I'll make that change 👍